package.json

```diff
@@ -54,6 +54,7 @@
     "angular": "^1.6.4",
     "angular-translate": "^2.11.0",
     "jquery-ui": "^1.12.1",
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
+    "egain-client-library": "^0.0.28"
```

Comment:
```
$ npm info egain-client-library
npm ERR! code E404
npm ERR! 404 'egain-client-library' is not in the npm registry.
```

Reply:
It's an internal Iib.

Reply:
So the chatbot is not an open source project anymore ? 🤔

Reply:
Well, not the livechat part I guess... Maybe I should add an option so that the feature is disabled by default, and can be enabled manually.